### DISCUSSION TITLE
Can't set Fallback SMS Number for 2FA

### DISCUSSION THREAD

**Discussion by rnstewart** [community/community#34485] [Other Feature Feedback, Questions, & Ideas]
I just set up 2FA for my Github account using the Microsoft Authenticator app.

I'm trying to set a fallback SMS number, but it's not working. I click on"Add Fallback SMS Number" on this page:

https://github.com/settings/two_factor_authentication/configure

And it brings up a dialog to enter a number. I enter my number (US area code) and click "Set Fallback", but nothing happens. A few minutes ago, it went as far as to say it was sending me an SMS code to confirm the number, but the SMS never arrived.

**Comment by NoahLyonsGeomagical** [comment]
Same issue here.

**Comment by alejandro-estrella** [comment]
Also having the same problem

**Comment by dragos-dobre** [comment]
Also having the same issue with France.

**Comment by Cloud-Forge** [comment]
same here

**Comment by kjspring** [comment]
Add me to the list where it is not working.

**Comment by TechMainul-dev** [comment]
Working with Bangladesh

**Comment by donaldbraman** [comment]
Same here.

**Comment by maghirardelli** [comment]
+1 I cannot set the fallback number either in the US. I click the "Set Fallback" button and it flicks to "Setting Fallback..." for a minute and then goes back to "Set Fallback" and it does not appear the number has been saved.

**Comment by gabe-dakota** [comment]
Also having this problem with a US number.

**Comment by OldRugger** [comment]
I am also having an issue - US number

**Comment by ngurnard** [comment]
Having the same issue with a US number
